spec/controllers/v2/security_guides_controller_spec.rb

```diff
@@ -32,6 +32,12 @@
     it_behaves_like 'paginable'
     it_behaves_like 'sortable'
     it_behaves_like 'searchable'
+
+    context 'with any read permission' do
+      before { stub_rbac_permissions(Rbac::REPORT_READ) }
+
+      it_behaves_like 'collection'
+    end
```

Comment:
Is there any point in having this test?

Reply:
It can be removed

Reply:
IMO for testing RBAC we either need separate tests independent from controllers, but I think it's better to keep as it is as it's fully declarative and I'd rather rely on QE to test declarative stuff.